using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md

```diff
@@ -333,7 +334,17 @@ func trackNotificationReceive(_ trackInfo: [String: String])
 **Example**
 
 ```swift
-ACPCampaignClassic.trackNotificationReceive(trackInfo[String:String])
+func application(_ application: UIApplication, didReceiveRemoteNotification launchOptions: [AnyHashable : Any], fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
+  if launchOptions != nil {
```

Comment:
You will mostly receive a warning for this nil check as `launchOptions` here is non-optional.